test/e2e/imagebundle/serve_bundle_oci_test.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2021 D2iQ, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build e2e
+
+package imagebundle_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/go-logr/logr/funcr"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/phayes/freeport"
+	"github.com/spf13/cobra"
+
+	"github.com/mesosphere/dkp-cli-runtime/core/output"
+
+	createbundle "github.com/mesosphere/mindthegap/cmd/mindthegap/create/bundle"
+	pushbundle "github.com/mesosphere/mindthegap/cmd/mindthegap/push/bundle"
+	servebundle "github.com/mesosphere/mindthegap/cmd/mindthegap/serve/bundle"
+	"github.com/mesosphere/mindthegap/docker/registry"
+	"github.com/mesosphere/mindthegap/images/httputils"
+	"github.com/mesosphere/mindthegap/test/e2e/imagebundle/helpers"
+)
+
+var _ = Describe("Serve Image Bundle with OCI artifacts", func() {
```

Comment:
Please rename this file and describe input - it's doing more than just `Serve`.

Reply:
Changed the filename and updated test descriptions.